package.json

```
@@ -0,0 +1,26 @@
+{
+  "name": "twentynineteen",
+  "version": "1.0.0",
+  "description": "Default WP Theme",
+  "bugs": {
+    "url": "https://github.com/WordPress/twentynineteen/issues"
+  },
+  "homepage": "https://github.com/WordPress/twentynineteen#readme",
+  "devDependencies": {
+    "autoprefixer": "^9.1.5",
+    "chokidar-cli": "^1.2.1",
+    "node-sass": "^4.9.3",
+    "postcss-cli": "^6.0.0",
+    "rtlcss": "^2.4.0"
+  },
+  "browserslist": [
+    "last 2 versions",
+    "ie >= 8"
+  ],
+  "scripts": {
+    "build:style": "node-sass style.scss style.css --output-style expanded && postcss -r style.css",
+    "build:style-editor": "node-sass style-editor.scss style-editor.css --output-style expanded && postcss -r style-editor.css",
+    "build": "run-p \"build:style\" \"build:style-editor\"",
```

this can just be 
```suggestion
    "build": "run-p \"build:*\"",
```

that way when you add new stuff like `build:something-else` it'll automatically get picked up by the wildcard.

see https://github.com/mysticatea/npm-run-all/blob/HEAD/docs/run-p.md